frontend/src/components/browse-sightings-page/mapPage.scss

```
@@ -0,0 +1,4 @@
+﻿.map {
```

It might not be the only map on the site!
```suggestion
﻿.sighting-map {
```